index.html

```diff
@@ -5,9 +5,9 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <h1>Welcome to Potionfy</h1>
+  <title>Welcome to Potionfy</title>
 </head>
 <body>
-  
+  <h1>Welcome to Potionfy! Enjoy your stay.</h1>
```

Comment:
Could you remove the `Enjoy your stay.` string, and make the `Welcome to Potionfy!` string red?

Reply:
Sure, can do!